views/js/previewer/plugins/content/scoringHighlighter.js

```diff
@@ -89,6 +70,58 @@ define([
                 })
             );
 
+            /**
+             * Gets the renges of the selection
+             * 
+             * @param {*} selection 
+             */
+            function getAllRanges(selection) {
+                const allRanges = [];
+
+                for (let i = 0; i < selection.rangeCount; i++) {
+                    allRanges.push(selection.getRangeAt(i));
+                }
+                return allRanges;
+            }
+
+            /**
+             * Erases highlights and updates highlightIndex
+             * 
+             * @param {Event} e - Click event
+             */
+            function clearHighlightAndSave(e) {
```

Comment:
Why not to make this function in the same style as `this.highlight` like `this.clearHighlightAndSave`?

Reply:
becuase they are not used outside init() and I thought its better to keep them scoped